tests/js/client/server_parameters/test-return-queue-time-header-on.js

```diff
@@ -162,7 +162,7 @@ function testSuite() {
       waitForPending(pendingAtStart, 1, n - 30);
 
       let result = arango.GET("/_api/version", { "x-arango-queue-time-seconds": "0.00001" });
-      assertEqual(412, result.code);
+      assertEqual(412, result.code, JSON.stringify(result));
```

Comment:
Nit (does not need to be changed): using `JSON.stringify()` here is fine, but not necessary. If the assertion goes wrong the `result` value would be pretty-printed anyway by `assertEqual`. At least it would not emit `[Object object]` or such.